src/runtime/internal/dom.ts

```diff
@@ -155,6 +155,7 @@ export function get_root_for_style(node: Node): ShadowRoot | Document {
 }
 
 export function append_stylesheet(node: ShadowRoot | Document, style: HTMLStyleElement) {
+	style.textContent = style.textContent || '/* empty */';
```

Comment:
we should have a comment explaining why this is necessary

Reply:
Yes. Yes we should.